bundle/Resources/config/services.yaml

```diff
@@ -93,7 +93,10 @@ services:
 
     ezrepoforms.field_definition.form_type:
         class: "%ezrepoforms.field_definition.form_type.class%"
-        arguments: ["@ezrepoforms.field_type_form_mapper.dispatcher", "@ezpublish.api.service.field_type"]
+        arguments:
+            - "@ezrepoforms.field_type_form_mapper.dispatcher"
+            - "@ezpublish.api.service.field_type"
+            - '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\Field\ContentFieldStrategy'
```

Comment:
Please create a service in Kernel which uses interface (API/SPI?) and use it here.